src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/tests/test_sf_commands.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import unittest
+import azure.cli.command_modules.servicefabric.custom
+from azure.cli.testsdk import ScenarioTest
+
+class ServiceFabricTest(ScenarioTest):
+    def test_list_storage_account(self):
+        cluster_list = self.cmd('az sf cluster list').get_output_in_json()
+        assert len(cluster_list) > 0
```

Comment:
Need more tests than this.
There are several new commands added and lots of custom logic to be tested.